src/Cli.Tests/EndToEndTests.cs

```diff
@@ -359,6 +359,35 @@ public void TestUpdateGraphQLPathRuntimeSettings(string path, bool isSuccess)
         Assert.AreEqual(isSuccess, isError == 0);
     }
 
+    /// <summary>
+    /// This test checks behavior of executing `dab configure --runtime.host.cors.origins {value}`
+    /// Validates that links provided for cors.origins result in DAB engine starting successfully.
+    /// Ensures that invalid links provided for Cors.Origins result in failed engine startup 
+    /// due to validation failure.
+    /// </summary>
+    [Ignore]
+    [DataTestMethod]
+    [DataRow("http://locahost1 https://localhost2", true, DisplayName = "Success in updating Host.Cors.Origins.")]
+    public void TestUpdateHostCorsOriginsRuntimeSettings(string path, bool isSuccess)
+    {
+        // Initialize the config file.
+        string[] initArgs = { "init", "-c", TEST_RUNTIME_CONFIG_FILE, "--host-mode", "development", "--database-type",
+            "mssql", "--connection-string", TEST_ENV_CONN_STRING };
+        Program.Execute(initArgs, _cliLogger!, _fileSystem!, _runtimeConfigLoader!);
+
+        Assert.IsTrue(_runtimeConfigLoader!.TryLoadConfig(TEST_RUNTIME_CONFIG_FILE, out RuntimeConfig? runtimeConfig));
+        Assert.IsNotNull(runtimeConfig);
+
+        // Act: Update the Path in the config file.
+        string[] runtimeArgs = { "configure", "-c", TEST_RUNTIME_CONFIG_FILE, "--runtime.host.cors.origins", path };
```

Comment:
So can we use "," as well as " "? please add that in the PR description as well as on the help text.